docs/REST API/rest-api.md

```diff
@@ -9178,6 +9178,8 @@ _Available in Fleet Premium._
 
 Install software (package or App Store app) on a macOS, iOS, iPadOS, Windows, or Linux (Ubuntu) host. Software title must have a `software_package` or `app_store_app` added to be installed.
 
+Note: Orbit only installs software it has been asked to install, but technically has access to all installer executables.
```

Comment:
Already approved this, but just realized it doesn't match the language we use elsewhere (we try not to say "Orbit")
```suggestion
> Note: Fleet's agent (fleetd) only installs software it has been asked to install, but technically has access to all installer executables.
```